google-cloud-bigquery/lib/google/cloud/bigquery/load_job.rb

```diff
@@ -961,6 +974,30 @@ def skip_leading= val
             @gapi.configuration.load.update! skip_leading_rows: val
           end
 
+          # Sets the encryption configuration of the destination table.
```

Comment:
Missing `##` comments header?

Reply:
Done in https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2016/commits/05f11c3800f6b4add6d44590e08dfb0fd1e34ac4